ContactDetailsApi/V2/Controllers/ContactDetailsController.cs

```diff
@@ -84,5 +88,19 @@ public async Task<IActionResult> CreateContact([FromBody] ContactDetailsRequestO
                 return BadRequest(e.ConstructResponse());
             }
         }
+
+        [HttpPatch]
+        [Route("{personId}/update/{contactDetailId}")]
```

Comment:
Maybe we should change the route to `/{contactDetailId}/person/{personId}`

Reply:
Yeah, I prefer this